config/application.rb

```diff
@@ -31,6 +31,12 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
 
+    # Enable the asset pipeline
+    config.assets.enabled = true
```

Comment:
Volgens mij is dit niet nodig. De default waarde is namelijk true en we zetten het nergens op false.